src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/FixLegacyResourceDesignerStep.cs

```diff
@@ -153,7 +156,11 @@ protected override void FixBody (MethodBody body, TypeDefinition designer)
 				if (idx >= 0) {
 					string key = line.Substring (idx + designerFullName.Length);
 					LogMessage ($"Looking for {key}.");
-					if (lookup.TryGetValue (key, out MethodDefinition method)) {
+					var found = lookup.TryGetValue (key, out MethodDefinition method);
```

Comment:
Why not instead:

```csharp
MethodDefinition method;
var found = lookup.TryGetValue (key, out method) || lookupCaseInsensitive.TryGetValue (key, out method);
if (found) { …
```

Reply:
Why not? because the `if (!found)` block is actually a useful place to emit a warning to indicate we're falling back to a case insensitive comparison!

Assuming we need to warn at all, but at least a message might be useful.

Reply:
I plan to add a message :D